component/web/security/src/main/java/org/exoplatform/web/login/LoginHandler.java

```diff
@@ -156,6 +156,9 @@ public boolean execute(ControllerContext context) throws Exception { // NOSONAR
     }
 
     String username = request.getParameter("username");
+    if (username != null) {
```

Comment:
this check is not needed

Reply:
needed in the case of loading the login page!